src/main/java/com/conveyal/r5/profile/PerTargetPropagater.java

```diff
@@ -44,7 +44,7 @@
     private PointSet targets;
 
     /** All modes for which we want to perform egress propagation through the street network. */
-    public final Set<StreetMode> modes;
+    public final Set<LegMode> modes;
```

Comment:
It feels like a significant change to be iterating over the LegModes instead of the StreetModes, but I suppose this is the only way we're going to see the _PARK and _RENT suffixes, and it would also enable better handling of the CAR_PARK option. I guess I've never really been comfortable with this suffix mode of designating additional behavior within a given on-street mode. It was inherited from OTP via Modeify, and it never really evolved into a coherent system.